scripts/events.py

```diff
@@ -1519,6 +1520,30 @@ def handle_apprentice_EX(self, cat):
 
             cat.experience += max(exp * mentor_modifier, 1)
 
+
+    def handle_outside_EX(self, cat):
+        if cat.status in ["loner", "rogue", "exiled", "kittypet", "lost"] and cat.age != 'kitten':
+            if cat.age == 'adolescent':
```

Comment:
What is the purpose of reducing EXP for kittypets, or for having an age-dependence on the amount of experience gained? If possible, we'd like reduce the number of nested if-else chains of this variety. 

In addition, the mix and max values should reference back to some value in game.config, to allow for easier tuning of values.

Reply:
I can see the benefit in reducing exp for kittypets, they probably get less warrior useful experience living that way, but yeah, I think these values should be handled in game.config- perhaps a base value and a kittypet value for these that are calculated and used instead of these if-else chains with hardcoded values for each possibility